include/cuco/static_set.cuh

```diff
@@ -460,6 +460,42 @@ class static_set {
   template <typename OutputIt>
   OutputIt retrieve_all(OutputIt output_begin, cuda_stream_ref stream = {}) const;
 
+  /**
+   * @brief Rebuilds the container.
+   *
+   * @note This function synchronizes the given stream. For asynchronous execution use
+   * `rehash_async`.
+   *
+   * @param stream CUDA stream used to get the number of inserted elements
+   */
+  void rehash(cuda_stream_ref stream = {});
+
+  /**
+   * @brief Rebuilds the container.
+   *
+   * @note This function synchronizes the given stream. For asynchronous execution use
+   * `rehash_async`.
+   *
+   * @param capacity New capacity of the container
+   * @param stream CUDA stream used to get the number of inserted elements
+   */
+  void rehash(size_type capacity, cuda_stream_ref stream = {});
+
+  /**
+   * @brief Rebuilds the container.
+   *
```

Comment:
docs explaining the difference between two overloads of `rehash`/`rehash_async`

Reply:
Addressed in [dcb0536](https://github.com/NVIDIA/cuCollections/pull/380/commits/dcb0536985e973ef9a377691f408603c8dfd0364) (I think)